prow/github/report/report.go

```diff
@@ -215,8 +240,20 @@ func parseIssueComments(pj prowapi.ProwJob, botName string, ics []github.IssueCo
 			previousComments = append(previousComments, latestComment)
 		}
 		latestComment = ic.ID
+		// Split the issue comment to obtain the list of SHAs written at the end of the comment
+		// after commentTag. tagSplit[1] cannot fail as we already know ic.Body contains the tag.
+		tagSplit := strings.Split(ic.Body, commentTag)
```

Comment:
This doesn't seem necessary, we can just `for _, line := range strings.Split(ic.Body, "\n") { ... }`, right?

AKA the exact location of the commit in this comment doesn't seem particularly important.

Reply:
hm, given now the first sentence writes the `pj.Spec.Refs.Pulls[0].SHA` commit on top of the issue comment, this means that this status report is for that particular commit and any different incoming `pj.Spec.Refs.Pulls[0].SHA` should clear comment, as long as we skip correctly old jobs in `Reptort()`. :thinking: 

which means the whole `<!-- list of SHAs at the end -->` might not be needed.
i will double check this.